app/views/api/backend_usages/edit.html.slim

```diff
@@ -1,11 +1,14 @@
 - content_for :page_header_title, 'Edit Backend Usage'
 
-= semantic_form_for @backend_api_config, url: admin_service_backend_usage_path(@service, @backend_api_config) do |form|
-  = form.inputs do
-    li
-      = form.label :backend_api_id, 'Backend'
-      = form.text_field :backend_api_id, class: 'text', value: @backend_api_config.backend_api.name, disabled: true
-    = form.input :path, hint: 'Changing the Public Path will only affect this Product.', label: 'Public Path'
+div class="pf-c-card"
+  div class="pf-c-card__body"
+    = semantic_form_for @backend_api_config, url: admin_service_backend_usage_path(@service, @backend_api_config),
+                                             html: { class: 'pf-c-form pf-m-limit-width' } do |form|
+      = form.input :backend_api_id, as: :patternfly_input,
+                                    label: 'Backend',
+                                    input_html: { readonly: true, value: @backend_api_config.backend_api.name }
+      = form.input :path, as: :patternfly_input, label: 'Public Path'
 
-  = form.actions do
-    = form.commit_button 'Update'
+      = form.actions do
+        = form.commit_button 'Update'
```

Comment:
I see the same case we saw at the New Provider Account PR, the bullet: 
![Captura de pantalla 2023-11-22 a las 17 24 16](https://github.com/3scale/porta/assets/72191439/77479098-4cd2-4f99-b682-25aed6aac5d3)

So I'd suggest the same solution:
```suggestion
        button class="pf-c-button pf-m-primary" type="submit" Update
```